includes/Core/Util/Requires_Javascript_Trait.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Trait Google\Site_Kit\Core\Util\Requires_Javascript_Trait
+ *
+ * @package   Google\Site_Kit\Core\Util
+ * @copyright 2019 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\Util;
+
+/**
+ * Trait to display no javascript fallback message.
+ *
+ * @since n.e.x.t
+ * @access private
+ * @ignore
+ */
+trait Requires_Javascript_Trait {
```

Comment:
@Shavindra it looks like the indentation here is a bit off. There's a .editorconfig file that can help you if you have the editorconfig extension.